hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -27,6 +27,38 @@
   ////////////////
 
   param_list: [
+    // Random netlist constants
+    { name:      "LcKeymgrDivInv",
```

Comment:
just a note here that i left a question to @cdgori on merging test and rma.
I think it SHOULD be okay.

Reply:
Confirmed in the other PR that merge dev/RMA/test_locked is OK for me.